Sources/MapboxMapsSwiftUI/MapCoordinator.swift

```diff
@@ -41,35 +41,45 @@ public final class MapCoordinator {
         }
     }
 
-    func update(from view: InternalMap) {
-        do {
-            try mapView.mapboxMap.performWithoutNotifying {
-                if let camera = view.camera {
-                    mapView.mapboxMap.setCamera(to: CameraOptions(cameraState: camera.wrappedValue))
-                }
+    func update(camera: CameraState?, deps: MapDependencies, colorScheme: ColorScheme) {
+        let mapboxMap = mapView.mapboxMap!
 
-                // TODO: This call can change the camera, but it won't be reflected on camera Binding.
```

Comment:
Additionnaly, solved this todo. It seems we don't have better option that just sync the camera state on next loop.